tests/mock_tables/dbconnector.py

```diff
@@ -34,6 +78,9 @@ class SwssSyncClient(mockredis.MockRedis):
     def __init__(self, *args, **kwargs):
         super(SwssSyncClient, self).__init__(strict=True, *args, **kwargs)
         db = kwargs.pop('db')
+        # Namespace is added in kwargs specifically for unit-test
+        # to identify the file path to load the db json files.
+        namespace = kwargs.pop('namespace')
         if db == 0:
             fname = 'appl_db.json'
```

Comment:
>fname [](start = 12, length = 5)

get the fname from config file?

Reply:
Added mock connect class to add filename and namespace in kwargs which can be retrieved in mock redis client class.